javascript/packages/orchestrator/src/orchestrator.ts

```diff
@@ -261,6 +261,38 @@ export async function start(
     const chainSpecContent = require(chainSpecFullPathPlain);
     client.chainId = chainSpecContent.id;
 
+    const parachainFilesPromiseGenerator = async (parachain: Parachain) => {
```

Comment:
This is just a reorder in the code, or I'm missing something?

Reply:
Its a reorder because we need to generate the parachain files, and since the relay chain we provided isnt raw, these codeblock does not run and the cumulus parachains fail to get created